src/yz_solrq_helper.erl

```diff
@@ -0,0 +1,225 @@
+%% -------------------------------------------------------------------
+%% Copyright (c) 2015 Basho Technologies, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(yz_solrq_helper).
+%% -compile([export_all,{parse_transform,pulse_instrument},{d,modargs}]). %%TODO: Dynamically add pulse. NOT PRODUCTION
+%% -compile({pulse_replace_module, [{gen_server, pulse_gen_server}]}).
+
+-include("yokozuna.hrl").
+
+%% api
+-export([start_link/1, status/1, status/2]).
+
+%% gen_server callbacks
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2,
+         terminate/2, code_change/3]).
+
+% solrq/helper interface
+-export([index_ready/3, index_batch/3]).
+
+-record(state, {qpid,
```

Comment:
qpid no longer used/http_pid never implemented.  Remove until we make dedicated HTTP connections.